From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Helvetica Volubi <suisuroru@blue-millennium.fun>
Date: Mon, 10 Aug 2026 03:32:54 +0800
Subject: [PATCH] Rebrand to Wind


diff --git a/src/main/java/io/papermc/paper/ServerBuildInfo.java b/src/main/java/io/papermc/paper/ServerBuildInfo.java
index 378c78b9321cbb96d95f0fb0d8c9c2ca2a283752..e32db891352c70b33daf9bf9e96ddb0534ee9f80 100644
--- a/src/main/java/io/papermc/paper/ServerBuildInfo.java
+++ b/src/main/java/io/papermc/paper/ServerBuildInfo.java
@@ -28,6 +28,13 @@ public interface ServerBuildInfo {
     Key BRAND_LOPHINE_ID = Key.key("lophinelabs", "lophine");
     // Lophine end
 
+    // Wind start
+    /**
+     * The brand id for Wind.
+     */
+    Key BRAND_WIND_ID = Key.key("flugobjekt", "wind");
+    // Wind end
+
     /**
      * Gets the {@code ServerBuildInfo}.
      *
